can't create table while using utf8 encoding in mysql

```
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]: java.sql.SQLSyntaxErrorException: Specified key was too long; max key length is 767 bytes
Query is : CREATE TABLE IF NOT EXISTS MUTES (UUID VARCHAR(256) PRIMARY KEY NOT NULL)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at org.mariadb.jdbc.internal.util.ExceptionMapper.get(ExceptionMapper.java:125)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at org.mariadb.jdbc.internal.util.ExceptionMapper.throwException(ExceptionMapper.java:69)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at org.mariadb.jdbc.MariaDbStatement.executeQueryEpilog(MariaDbStatement.java:242)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at org.mariadb.jdbc.MariaDbStatement.executeInternal(MariaDbStatement.java:270)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at org.mariadb.jdbc.MariaDbStatement.execute(MariaDbStatement.java:283)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.zaxxer.hikari.pool.ProxyStatement.execute(ProxyStatement.java:95)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.zaxxer.hikari.pool.HikariProxyStatement.execute(HikariProxyStatement.java)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at io.github.hsyyid.essentialcmds.utils.Utils.execute(Utils.java:586)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at io.github.hsyyid.essentialcmds.utils.Utils.readMutes(Utils.java:613)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at io.github.hsyyid.essentialcmds.EssentialCmds.onServerInit(EssentialCmds.java:199)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at org.spongepowered.common.event.listener.GameInitializationEventListener_EssentialCmds_onServerInit3.handle(Unknown Source)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at org.spongepowered.common.event.RegisteredListener.handle(RegisteredListener.java:95)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:354)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:395)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at org.spongepowered.mod.SpongeMod.onStateEvent(SpongeMod.java:205)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at java.lang.reflect.Method.invoke(Method.java:497)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.google.common.eventbus.EventBus.post(EventBus.java:275)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:235)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:213)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at java.lang.reflect.Method.invoke(Method.java:497)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at com.google.common.eventbus.EventBus.post(EventBus.java:275)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at net.minecraftforge.fml.common.LoadController.redirect$onPost$0(LoadController.java:53)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:142)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:774)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at net.minecraftforge.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:107)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at net.minecraftforge.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:333)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:214)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:431)
[14:27:57] [Server thread/INFO] [STDERR]: [io.github.hsyyid.essentialcmds.utils.Utils:execute:593]:     at java.lang.Thread.run(Thread.java:745)
[14:27:57] [Server thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:634]: Caused by: org.mariadb.jdbc.internal.util.dao.QueryException: Specified key was too long; max key length is 767 bytes
Query is : CREATE TABLE IF NOT EXISTS MUTES (UUID VARCHAR(256) PRIMARY KEY NOT NULL)
[14:27:57] [Server thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:634]:    at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.getResult(AbstractQueryProtocol.java:938)
[14:27:57] [Server thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:634]:    at org.mariadb.jdbc.internal.protocol.AbstractQueryProtocol.executeQuery(AbstractQueryProtocol.java:604)
[14:27:57] [Server thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:634]:    at org.mariadb.jdbc.MariaDbStatement.executeInternal(MariaDbStatement.java:261)
[14:27:57] [Server thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:634]:    ... 39 more

```

mysql primary key must shorter than 767 bytes.But if using utf8, varchar(256) will use 768bytes to storage.
I can build table manually by "CREATE TABLE IF NOT EXISTS MUTES (UUID VARCHAR(256) PRIMARY KEY NOT NULL) charset=ascii",but essentialcmds can't init currectly.

--- a/src/main/java/io/github/hsyyid/essentialcmds/utils/Utils.java
+++ b/src/main/java/io/github/hsyyid/essentialcmds/utils/Utils.java
@@ -187,7 +187,7 @@ public class Utils
 
 				DataSource datasource = sql.getDataSource("jdbc:mysql://" + host + ":" + port + "/" + database + "?user=" + username + "&password=" + password);
 
-				String executeString = "CREATE TABLE IF NOT EXISTS MUTES " + "(UUID VARCHAR(256) PRIMARY KEY NOT NULL)";
+				String executeString = "CREATE TABLE IF NOT EXISTS MUTES " + "(UUID CHAR(36) PRIMARY KEY NOT NULL)";
 				execute(executeString, datasource);
 
 				executeString = "DELETE FROM MUTES WHERE UUID='" + playerUUID.toString() + "';";

